swagger.json

```diff
@@ -1691,6 +1741,64 @@
                 }
             }
         },
+        "/fleet/vehicles/{vehicle_id}/locations": {
+          "parameters": [
+            {
+              "$ref": "#/parameters/accessTokenParam"
+            },
+            {
+              "name": "vehicle_id",
+              "in": "path",
+              "required": true,
+              "description": "ID of the vehicle with the associated routes.",
+              "type": "integer",
+              "format": "int64"
+            },
+            {
+              "name": "startMs",
+              "description": "Time in millis for the start of the query (cannot exceed 1 hour)",
+              "required": true,
+              "in": "query",
+              "type": "integer"
+            },
+            {
+              "name": "endMs",
+              "description": "Time in millis for the end of the query (cannot exceed 1 hour)",
+              "required": true,
+              "in": "query",
+              "type": "integer"
+            }
+          ],
+          "get": {
+            "tags": [
+              "Default",
+              "Fleet"
+            ],
+            "summary": "/fleet/vehicles/{vehicle_id:[0-9]+}/locations",
+            "description": "Fetch locations for a given vehicle between a start/end time. The maximum query duration is one hour.",
+            "operationId": "getVehicleLocations",
+            "consumes": [
+              "application/json"
+            ],
+            "produces": [
+              "application/json"
+            ],
+            "responses": {
+              "200": {
+                "description": "Returns locations for a given vehicle between a start/end time, with a maximum query duration of one hour.",
```

Comment:
Is the max query duration 30 minutes?

Reply:
It is 1 hour for the single vehicle case, but 30 minutes for all vehicles